api/src/routes/messenger.ts

```diff
@@ -70,6 +70,9 @@ messenger.post("/camel", bodyParser.json(), async function (req, res) {
         case "Purge":
             await QueueManager.getInstance().performIndexOperation(pid, "delete");
             break;
+        case "Follow":
```

Comment:
I don't think you need this additional break statement. You can just move it one line up before the previous break statement.

Reply:
That's true, though there may be a stylistic advantage to keeping it separated like this (e.g. if we eventually decide to add functionality to this case, we don't want the previous case to drop through into it... and that might be an easy mistake to make otherwise).